test/views/RemoveSavedDeclarationsViewSpec.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import java.time.{LocalDateTime, ZoneOffset}
+
+import base.ExportsTestData
+import base.ExportsTestData._
+import forms.declaration.ConsignmentReferences
+import forms.{Ducr, RemoveDraftDeclaration}
+import helpers.views.declaration.CommonMessages
+import models.{DeclarationStatus, ExportsDeclaration}
+import org.jsoup.nodes.Element
+import play.api.data.Form
+import play.twirl.api.Html
+import views.declaration.spec.ViewSpec
+import views.html.remove_declaration
+import views.tags.ViewTest
+
+@ViewTest
+class RemoveSavedDeclarationsViewSpec extends ViewSpec with CommonMessages {
+
+  val title: String = "saved.declarations.remove.title"
+  val ducr: String = "saved.declarations.ducr"
+  val dateSaved: String = "saved.declarations.dateSaved"
+
+  private val removeSavedDeclarationsPage = app.injector.instanceOf[remove_declaration]
```

Comment:
I know actual approach, but you can create view manually, in `Stubs` you have mainTemplate for views, but with this we will be checking error keys, not messages from specific key